src/NSwag.CodeGeneration.CSharp/Models/CSharpOperationModel.cs

```diff
@@ -58,7 +58,7 @@ public class CSharpOperationModel : OperationModelBase<CSharpParameterModel, CSh
                 {
                     parameters = parameters
                         .OrderBy(p => p.Position ?? 0)
-                        .OrderBy(p => !p.IsRequired)
+                        .ThenBy(p => !p.IsRequired)
```

Comment:
I think this should not be changed, as all required parameters must come first for correctly working null defaults...

Reply:
Ok, I'll leave it as it was, but as far as I understand the ThenBy approach is the recommended way.
Here's more on that: https://stackoverflow.com/questions/3760001/linq-orderby-versus-thenby

Reply:
Done.

Reply:
That's true. However, if we want t keep the existing behavior, `.OrderBy(p => p.Position ?? 0).OrderBy(p => !p.IsRequired)` would need to be replaced by `.OrderBy(p => !p.IsRequired).ThenBy((p => p.Position ?? 0))` or the sorting order would change because the current IsRequired sorting is taking precedence since it's being applied after the Position sorting.